pkg/build/push.go

```diff
@@ -261,15 +262,17 @@ func (bi *Instance) CheckReleaseBucket() error {
 
 	client, err := storage.NewClient(context.Background())
 	if err != nil {
-		return errors.Wrap(err,
+		return fmt.Errorf(
 			"fetching gcloud credentials, try running "+
-				`"gcloud auth application-default login"`,
+				`"gcloud auth application-default login"`+
+				" %w",
```

Comment:
nit: maybe add the `:` to match the others

Reply:
Thanks for pointing @cpanato. Fixed! 🙂